### PR TITLE
Issue #5118 - Documentation for L.Map.FitBounds is incorrect

### LINKED ISSUE #5118: Documentation for L.Map.FitBounds is incorrect?
- [x] I'm reporting a bug, not asking for help
- [x] I've looked at the [documentation](http://leafletjs.com/reference-1.0.0.html) to make sure the behavior is documented and expected
- [x] I've searched through the issues to make sure it's not yet reported

FitBounds documentation describes that options parameter is not optional, I'm not sure this is the case as the codes tell me otherwise, am I wrong?

P.S. you have a spelling mistake in the report issue template for the word "behaviour"... :-)

### PR DESCRIPTION
**Pull request by rodrijuarez**
This resolves #5118

### CODE REVIEW

File: docs/examples.md [hunk 1]
```diff
@@ -9,103 +9,115 @@ bodyclass: examples
 Every tutorial here comes with step-by-step code explanation and is easy enough even for beginner JavaScript developers.
 
 
-{% include tutorial_link.html
+{% raw %}
```
Comment (perliedman) [inline docs/examples.md]: These changes look unrelated to the actual topic of the PR. Please only change things relevant to the thing your trying to fix.

File: docs/reference-1.0.0.html [hunk 2]
```diff
@@ -1019,7 +1019,7 @@ <h4 id='map-methods-for-modifying-map-state'>Methods for modifying map state</h4
 </td>
 	</tr>
 	<tr id='map-setzoom'>
-		<td><code><b>setZoom</b>(<nobr>&lt;Number&gt;</nobr> <i>zoom</i>, <nobr>&lt;<a href='#zoom/pan-options'>Zoom/pan options</a>&gt;</nobr> <i>options</i>)</nobr></code></td>
+		<td><code><b>setZoom</b>(<nobr>&lt;Number&gt;</nobr> <i>zoom</i>, <nobr>&lt;<a href='#zoom/pan-options'>Zoom/pan options</a>&gt;</nobr> <i>options?</i>)</nobr></code></td>
```
Comment (perliedman) [inline docs/reference-1.0.0.html]: These are changes in the _compiled_ HTML docs - any changes here will be overwritten next time we compile the documentation.

The real documentation is inside the code, see [CONTRIBUTING.md](https://github.com/Leaflet/Leaflet/blob/master/CONTRIBUTING.md#improving-documentation) on improving docs.

**Comment by IvanSanchez** [comment]
Superseded by 1f4c1b9 - there is no need to change the compiled documentation files, nor the templates.
